index.js

```diff
@@ -8,6 +9,8 @@ config.load(function(error, config) {
   var server = require('./lib/proxy').server
   var log = require('./lib/logger').getLogger()
 
+  server.timeout = ms(config.serverTimeout || '10m');
```

Comment:
Will this time out the connection we are proxying to as well? Or just drop the request to the proxy and leave the connection to the backend open?